test/tools/jsontests/vm.cpp

```diff
@@ -263,7 +263,7 @@ eth::OnOpFunc FakeExtVM::simpleTrace() const
         /*add the stack*/
         Array a_stack;
         for (auto i : vm.stack())
-            a_stack.push_back((string)i);
+            a_stack.push_back(i.convert_to<string>());
```

Comment:
Presumably these changes are dependent on the new version of boost? If not perhaps we should include them in a different PR?

Reply:
The previous way of converting multiprecision int to string is not working in the latest Boost. I have not found any documentation about this braking change nor what is the recommended way of doing that, but `.convert_to()` works.

I missed some cases, will fix it later.